pkg/controller/installation/core_controller.go

```diff
@@ -426,6 +427,16 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 
 	ctx := context.Background()
 
+	// Get the installation object if it exists so that we can save the original
+	// status before we merge/fill that object with other values.
+	instance := &operator.Installation{}
+	if err := r.client.Get(ctx, utils.DefaultInstanceKey, instance); err != nil && apierrors.IsNotFound(err) {
```

Comment:
Do we really need this extra get? Couldn't we use the status on the object that comes from GetInstallation?

Reply:
The main reason we are doing this here is because GetInstallation actually does two things: it gets the Installation then it merges defaults and such into that Installation before returning it. I don't think changing the signature of GetInstallation makes sense (since other usages of GetInstallation don't need the original Instance).

The Installation returned by GetInstallation will have its status already set to the default.